src/installer/installer.go

```diff
@@ -436,9 +437,46 @@ func (i *installer) waitForControlPlane(ctx context.Context) error {
 		return err
 	}
 
+	i.waitForETCD(ctx, kc)
```

Comment:
where is the one minute wait that we have right now? 
This PR only added functionality, if i remember correctly @eranco74 you wanted to replace the one minute wait with a new logic no? 
I understand that we will need both because we have older version as well, just was thinking that it should be in the same place.

Reply:
Here is the one minute delay we currently have
https://github.com/openshift/assisted-installer/blob/adf62c69759137c6d351bcacc74464c867299adc/src/installer/installer.go#L202

Reply:
Do you all think we still need this delay?

Reply:
if we have this code i think that we can skip this minute, @eranco74 what do you think?